index.bs

```diff
@@ -1893,9 +1893,9 @@ The <dfn export for=commands>session.end</dfn> command ends the current
    </dd>
    <dt>Result Type</dt>
    <dd>
-      <pre class="cddl">
+      <code>
```

Comment:
maybe we should be defining a type for each return type:

```
session.EndResult = EmptyResult
```

so that one can easily find the matching return type event if it is empty.

Reply:
Not sure who you're asking ;) That seems good to me but I'm not sure I understand the consequences this could have on implementations.

Would you like me to update the PR to add types for each return type as suggested?

Reply:
We can if it helps something. The reason we didn't is that it seemed like extra boilerplate to have to specifically define a novel return type for every command, even if it's just an alias for an existing type (but we still want the spec to say what it returns).

Reply:
I think we can land this CL though to unblock the CDDL extraction.